app/src/app/components/header/NavMenu.tsx

```diff
@@ -0,0 +1,81 @@
+import { hasPacketRunPermission, hasUserManagePermission } from "../../../lib/auth/hasPermission";
```

Comment:
This is a merge of LeftNav (which is no longer on the left, hence, 'NavMenu'), with NavMenuMobile. Both of those were making use of the same logic to process 'NavItems' into a list of nav items that the user had permission to see, so I thought it would make more sense to handle that just once, in a component that concerns itself with that logic, and then the markup follows from that.